### ISSUE TITLE
Hashes of java implementation don't match output of hashids.js

### ISSUE THREAD

**Issue by thmarti** [peet/hashids.java#4]
I'd say it renders this library pretty much useless!
# Java

Hashids hashids = new Hashids("this is my salt");
String hash = hashids.encrypt(12345);
=> hash: **ryBo**
# JavaScript

var hashids = new Hashids("this is my salt"),
hash = hashids.encrypt(12345),
=> hash: **NkK9**

**Comment by peet** [comment]
Yeah, unfortunately an incompatible change was made to the JavaScript version after I ported it to Java. Re-porting is on my todo list, but it might not be for a little while yet.
